### ISSUE TITLE
tapping FocusExposureSwitchWidget only works once

### ISSUE THREAD

**Issue by terrysmay** [dji-sdk/Mobile-SDK-Android-V5#69]
Once I toggle the the FocusExposureSwitchWidget to use Focus mode metering ( the icon looks like a box, instead of the golden circle) I am not able to toggle back to Spot Metering.

here are some related Logcat lines:

```
2023-02-03 14:06:13.740 14709-14709 FocusExpoSwitchWidget   com.dji.sampleV5.aircraft            E  (lambda$errorConsumer$0:56) switchControlMode: ErrorImp{errorType='CORE', errorCode='SYSTEM_ERROR', innerCode='', description='null', hint='error code = -7'} 

2023-02-03 14:06:14.622 14709-14709 FocusExpoSwitchWidget   com.dji.sampleV5.aircraft            E  (lambda$errorConsumer$0:56) switchControlMode: ErrorImp{errorType='CORE', errorCode='SYSTEM_ERROR', innerCode='', description='null', hint='error code = -7'} 

```

How to reproduce:
 * open default layout tap the FocusExposureSwitchWidget in the top right
 * tap the camera view to meter an area, observe a box is placed where you tap
 * tap the FocusExposureWidget again
 * observe that nothing happens on the screen
 * observe lines of failure in logcat

I have tried this on the M30 and M3E with a freshly compilied sample app

**Comment by DJI-William** [comment]
It nevers work on my side, I will raise a bug ticket for it.

**Comment by dji-dev** [comment]
This issue will be fixed in 5.5 or other versions. The engineers have finished development.
